### PR TITLE
feat(gio-menu): add keyboard accessibility in collapse mode

### PR DESCRIPTION
**Pull request by gcusnieux**
**Issue**
https://graviteecommunity.atlassian.net/browse/APIM-291

---
📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-voxotcraiz.chromatic.com)

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-29b9644
# or
yarn add @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-29b9644
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-29b9644
# or
yarn add @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-29b9644
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-ee0ed7d
# or
yarn add @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-ee0ed7d
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-ee0ed7d
# or
yarn add @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-ee0ed7d
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-abecc01
# or
yarn add @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-abecc01
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-abecc01
# or
yarn add @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-abecc01
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-8ccd1fb
# or
yarn add @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-8ccd1fb
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-b31b472
# or
yarn add @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-b31b472
```

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-item/gio-menu-item.component.ts [hunk 1]
```diff
@@ -40,21 +42,36 @@ export class GioMenuItemComponent {
...
+      .subscribe();
   }
 
   @HostListener('keydown', ['$event'])
-  public onKeydownHandler(event: KeyboardEvent): void {
-    if (event.key === ' ' || event.key === 'Enter') {
-      this.gioMenuItem?.nativeElement.click();
-    }
+  public onKeydownHandler($event: KeyboardEvent): void {
+    of($event.key === 'Enter' && $event.target)
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-item/gio-menu-item.component.ts:62]: Should we not unsubscribe ?

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-a81f38c
# or
yarn add @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-a81f38c
```

**Review by ThibaudAV — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-0a5e682
# or
yarn add @gravitee/ui-particles-angular@4.12.0-291-keyboard-accessibility-0a5e682
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-0a5e682
# or
yarn add @gravitee/ui-policy-studio-angular@4.12.0-291-keyboard-accessibility-0a5e682
```

**Pull request merged**